contracts/BSCValidatorSet.sol

```diff
@@ -255,7 +302,15 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     }
   }
 
-  function jailValidator(Validator memory v) internal returns (uint32) {
+  function jailValidator(address consensusAddress) external onlyStakeHub {
+    uint256 index = currentValidatorSetMap[consensusAddress];
+    if (index==0 || currentValidatorSet[index-1].jailed) {
```

Comment:
The check here is duplicated with `_jailValidator`

Reply:
Removed